srcpkgs/ulogd/files/ulogd.logrotate

```diff
@@ -1,9 +1,9 @@
-/var/log/ulog.log /var/log/ulog.pcap {
-    missingok
+/var/log/ulogd/* {
     compress
+    missingok
+    notifempty
     sharedscripts
-    create 640 _ulogd adm
     postrotate
-	sv hup ulogd
+        sv hup ulogd
```

Comment:
not sure what happened here, but please try to match tabs/spaces.

Reply:
The logrotate file was using spaces and tabs so I changed it to spaces only. I reverted this change.